### PR TITLE
GDScript: add randn()

### PR DESCRIPTION
**Pull request by ghost**
...which would allow to get random numbers normally distributed around a given mean and deviation:

```
	var data = []
	for i in range(200):
		data.append(0)
	for i in range(1000):
		var x = randn(100, 10)
		data[min(max(0, x), 199)] += 1

	var s = 4
	for i in range(len(data)):
		var y = 300
		var p = Vector2(i * s, y)
		draw_line(p, p + Vector2(0, -data[i] * s), Color(0.9, 0.5, 0.6), 2)
```

dist

### CODE REVIEW

**Comment by akien-mga** [comment]
I think it would be more interesting to get an instanceable `Random` class with such methods, as described in #7989 (especially towards the end).

**Comment by DeadlyLampshade** [comment]
I agree, I would rather a system that utilizes an instanceable random class, it would be useful for storing and utilizing RNG with different seeds.

**Comment by poke1024** [comment]
https://github.com/godotengine/godot/pull/7989 is very good module. And, if that module gets added as core functionality, this PR is definitely redundant in its current implementation.

Still, I wonder if there shouldn't still be a global `randn()` that internally calls that module with one global default RNG instance.

I personally won't usually use multiple RNG instances in a single threaded environment; I mean, a RNG generates random numbers, so the point of having different streams of randomness seems to me solely an issue of debugging for reproducibility. Having an interface that expects me to use RNG instances in all cases feels overly complex to me; you need to make your RNG instance variables available in all places where you want to use it, which can become much more tedious than simply writing a global `randn()` everywhere you want it.

I'm not arguing against a `Random` class, I'm just arguing for a simple global baseline interface in addition with a default RNG instance, where something like `randn()` could still have a place.

File: doc/classes/@GDScript.xml [hunk 1]
```diff
@@ -776,6 +776,16 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="randn">
+			<return type="float">
+			</return>
+			<description>
+				randn(m, s) returns a normally distributed random number with m as mean as s as standard deviation.
+				[codeblock]
+				randn(10.0, 5.0) # returns 7.23421
```
Comment (bojidar-bg) [inline doc/classes/@GDScript.xml]: Obligatory xkcd: 😃 

(Yea, I know, it is the same for the randf documentation)

Comment (bfloch) [reply doc/classes/@GDScript.xml]: http://dilbert.com/strip/2001-10-25

**Pull request closed without merging**